astropy/io/ascii/basic.py

```diff
@@ -16,29 +16,32 @@
 from . import core
 
 class BasicHeader(core.BaseHeader):
-    '''Basic table Header Reader
+    """
```

Comment:
I also prefer double quotes, for consistency if nothing else.  Agree it's largely irrelevant :)